lisa/variable.py

```diff
@@ -213,7 +214,25 @@ def _load_from_file(
         raise LisaException("variable file must be dict")
 
     for key, raw_value in raw_variables.items():
-        results.update(load_from_variable_entry(key, raw_value, is_secret=is_secret))
+        try:
+            entry = convert_to_variable_entry(raw_value, is_secret=is_secret)
+            raw_value = entry.value
+            is_secret = is_secret or entry.is_secret
+            is_case_visible = entry.is_case_visible
+            mask = entry.mask
```

Comment:
How about making scheme of variables from file similar how it is in runbook ?
Currently this supports following. 
```YAML
var1: value1
var2:
  value: value2
  
```
But can have similar schema as used in runbook.
```YAML
- name: var1
  value: value1
```

Reply:
In the variable file, the current schema is still supported. It's not changed this time. If it needs to have the second format, it can be a standard runbook, and use include to use it.

Reply:
The file format was designed to compatible with LISAv2. LISAv2 uses xml, but it's like <name>value</name> format. It's a simpler format, than LISAv3 used. So I prefer to remove support for file more than use the same format as runbook. It's duplicate with the included runbook.